kyo-bench/src/main/scala/kyo/bench/CollectParBench.scala

```diff
@@ -10,7 +10,9 @@ class CollectParBench extends Bench.ForkOnly(Seq.fill(1000)(1)):
     override def kyoBenchFiber() =
         import kyo.*
 
-        Async.parallel(kyoTasks)
+        // TODO inference issue
+        val x = Async.parallel(kyoTasks)
+        x
```

Comment:
The compilation fails reporting that `Abort[Throwable]` isn't a context effect. It seems a bad interaction with effect widening. It's pre-existing, I'll follow up on this separately.